app/mixins/message-wall-post-notification.js

```diff
@@ -0,0 +1,33 @@
+import Mixin from '@ember/object/mixin';
+import { getMessageWallOwner, getPossiblyAnonActorName } from '../utils/message-wall';
+
+export default Mixin.create({
+  /**
+  * Constructs a localized message wall post notification body
+  * @param {Ember.Object} model
+  * @returns {string}
+  */
+  getMessageWallPostBody(model) {
+    let wallOwner = model.get('metadata') && model.get('metadata.username');
```

Comment:
I think the model getter is safe for `undefined` so you only need `let wallOwner = model.get('metadata.username');` by itself.